### PR TITLE
#195 avoid tight coupling

### LINKED ISSUE #195: Avoid tight coupling
Internal implementation shouldn't be aware of its outside usage. It should be separated and not coupled, this rule applies for the naming as well. 
Let's take a below example, `openDetails` directly suggest what action should be performed, but it's a responsibility of external component. If we would like to play a video or send tracking event on click, then we would need to change a name to not confuse rest of the team. Trivial thing but can involve to really confusing thing.  
```
@Composable
private fun PartyTile(
    party: Party,
    openDetails: (Int) -> Unit,
) {
    Card(
        modifier = Modifier.clickable {	
            openDetails(party.id)
	},
   ....
```

### PR DESCRIPTION
**Pull request by kmaslowiec**

### CODE REVIEW

**Review by bartek977 — APPROVED** [review]

File: app/src/main/kotlin/com/intive/picover/parties/view/PartiesScreen.kt [hunk 1]
```diff
@@ -84,13 +84,13 @@ private fun LoadedContent(parties: List<Party>, openDetails: (Int) -> Unit) {
 @Composable
 private fun PartyTile(
 	party: Party,
-	openDetails: (Int) -> Unit,
+	onPartyClick: (Int) -> Unit,
```
Comment (MarekMacko) [inline app/src/main/kotlin/com/intive/picover/parties/view/PartiesScreen.kt]: Here, I would suggest to simplify it to standalone `onClick`

Comment (kmaslowiec) [reply app/src/main/kotlin/com/intive/picover/parties/view/PartiesScreen.kt]: Updated. Thank you.

**Review by MarekMacko — APPROVED** [review]

File: app/src/main/kotlin/com/intive/picover/parties/view/PartiesScreen.kt (hunk 1 above)
Comment (MarekMacko) [reply app/src/main/kotlin/com/intive/picover/parties/view/PartiesScreen.kt]: You're welcome.

**Review by marcinWisnia — APPROVED** [review]

**Review by BorysChajdas — APPROVED** [review]

**Review by michal-kucznerowicz — APPROVED** [review]

**Comment by MarekMacko** [comment]
@kmaslowiec Is it ready to be merged?

**Comment by kmaslowiec** [comment]
> @kmaslowiec Is it ready to be merged?

Once I update it I will merge it.

**Pull request merged**